### DISCUSSION TITLE
Away from local network connection?

### DISCUSSION THREAD

**Discussion by Ringston** [unknownskl/xbox-xcloud-client#1108] [Q&A] [answered]
Hello,

This app is perfectly optimized and works awesome on my M3 Pro Macbook when I'm connecting via local network. However, I get stuck with a continuous buffer when trying to connect away from home. Is this app only meant for local use, and if not, are there any troubleshooting measures I can take to try and fix this? Thanks in advance.

**Comment by rackerby** [comment, marked as answer]
See this thread: https://github.com/unknownskl/greenlight/issues/593

TLDR; you need to forward UDP port 9002 to your Xbox console.
